docs/changelog.rst

```diff
@@ -12,6 +12,24 @@ schemes recommended by the Python Packaging Authority.
 
 .. towncrier release notes start
 
+Twine 3.5.0 (2021-11-02)
+------------------------
+
+Features
+^^^^^^^^
+
+- Show more helpful messages for invalid passwords. (`#815 <https://github.com/pypa/twine/issues/815>`_)
+- Allow the ``--skip-existing`` option to work with GCP Artifact Registry. (`#816 <https://github.com/pypa/twine/issues/816>`_)
```

Comment:
I missed this in the original PR; our [changelog docs](https://twine.readthedocs.io/en/latest/contributing.html#changelog-entries) ask that entries link to the PR, not the issue.

```suggestion
- Allow the ``--skip-existing`` option to work with GCP Artifact Registry. (`#823 <https://github.com/pypa/twine/issues/823>`_)
```